capella2polarion/converters/element_converter.py

```diff
@@ -33,6 +35,14 @@
 logger = logging.getLogger(__name__)
 C2P_IMAGE_PREFIX = "__C2P__"
 JINJA_RENDERED_IMG_CLS = "jinja-rendered-image"
+ARCHITECTURE_LAYERS: dict[str, str] = {
+    "common": "Common",
+    "oa": "Operational Analysis",
+    "sa": "System Analysis",
+    "la": "Logical Architecture",
+    "pa": "Physical Architecture",
+    "epbs": "EPBS",
+}
```

Comment:
I think it would be better to have this as enum defined in Polarion, wouldn't it? This way we could just post the actual values like `oa` to Polarion and there it can be handled including localization.